17_differential_abundance.Rmd

```diff
@@ -6,13 +6,312 @@ chapterPreamble()
 ```
 
 
+## Introduction to differential abundance analysis (DAA)
+
+*What is DAA and why do we do it?*
+
+In this section we will work with one of the [openly available datasets](https://rdrr.io/github/FelixErnst/mia/man/mia-datasets.html)in mia to illustrate how to perform differential abundance analysis (DAA). DAA aims to identify differences in taxon abundance between two or more groups (e.g. treatment vs control) for each taxon of a sample. This can be performed at any phylogenetic level but we will work at Genus level abundances in this section. We perform DAA to identify biomarkers and/or gain understanding of a complex system by looking at its isolated components. For example, identifying that a bacterial taxon is different between e.g. a patient group with disease *X* vs a healthy control group might lead to important insights into the pathophysiology. Changes in the microbiota might be causal or a consequence of the disease. Either way, it can help to understand the system as a whole. Be aware that this approach has also been critisized [recently](https://arxiv.org/abs/2104.07266).
```

Comment:
1) This website... Should it be https://microbiome.github.io/mia/reference/mia-datasets.html

2) ia/man/mia-datasets.html)in mia to illusrate...
There is no space between web address and "in"

Reply:
And other too long lines also...? You can check other files, how wide the line should be